templates/archives/stories/digital_humanities.jinja2

```diff
@@ -0,0 +1,43 @@
+{% extends 'archives/story.jinja2' %}
+
+{% block story_title %}
+Digital Humanities: an Enduring Legacy
+{% endblock %}
+
+{% block story_teaser %}
+A time without digital humanities is better known as a time without computers. Even at its
+    earliest, when the hours of computation were meticulously counted and rationed, the
+    investment of technology in humanism thrived.
+{% endblock %}
+
+{% block story_body %}
+    <p> Depending on how aware one is regarding the early <a href="/archives/story/debugging">perils of debugging</a>,
+    it's not difficult to understand exactly how valuable even a single hour on a computer was for
```

Comment:
Maybe this can be rephrased by explaining how expensive it was to use computers at the time of these archives. You can say how debugging was a particular pain point, but I think the argument that computers being expensive to use and still committing to doing humanities research is really spectacular, so I would highlight the argument more than the details.